packages/devtools_app/lib/src/shared/http/http_request_data.dart

```diff
@@ -55,14 +56,33 @@ class DartIOHttpRequestData extends NetworkRequest {
   HttpProfileRequestRef _request;
 
   final int wrapperId;
+  bool isOutStanding = false;
 
-  Future<void> getFullRequestData() {
-    return serviceManager.service!
-        .getHttpProfileRequest(
-          _request.isolateId,
-          _request.id.toString(),
-        )
-        .then((updated) => _request = updated);
+  /// A notifier that changes when the request data, or it's response body
+  /// changes.
+  ValueNotifier<int> updateCount = ValueNotifier<int>(0);
```

Comment:
Should this be modifiable outside of this class? If not, you'll want to make this private and expose a public `ValueListenable<int> get updateCount => _updateCount` instead.